cluster.tf

```diff
@@ -22,6 +22,10 @@ resource "aws_eks_cluster" "this" {
     public_access_cidrs     = var.cluster_endpoint_public_access_cidrs
   }
 
+  kubernetes_network_config {
+    service_ipv4_cidr = var.service_ipv4_cidr != "" ? var.service_ipv4_cidr : null
```

Comment:
wouldn't be simpler if we  define `var.service_ipv4_cidr` default to `null` and then just write:

```hcl
kubernetes_network_config {
    service_ipv4_cidr = var.service_ipv4_cidr
}
```